Prox/Prox/Utilities/PlaceUtilities.swift

```diff
@@ -21,6 +22,43 @@ struct PlaceUtilities {
         }
     }
 
+    static func sort(places: [Place], byTravelTimeFromLocation location: CLLocation, ascending: Bool = true, completion: @escaping ([Place]) -> ()) {
+        PlaceUtilities.getTravelTimes(forPlaces: places, fromLocation: location).upon { result in
+            let sortedPlaces = places.sorted { (placeA, placeB) -> Bool in
+                guard let placeATravelTimes = PlaceUtilities.lastTravelTimes(forPlace: placeA),
+                    let placeBTravelTimes = PlaceUtilities.lastTravelTimes(forPlace: placeB) else {
+                        return false
+                }
+
+                let placeAETA = placeATravelTimes.getShortestTravelTime()
+                let placeBETA = placeBTravelTimes.getShortestTravelTime()
+
+                guard ascending else {
```

Comment:
nit: I might just be a Swift noob but this is confusing as a `guard`.

(But I might also argue that having what amounts to an `if` statement with reverse semantics can also confusing! :P But I like the intent of showing early returns)